scene/gui/text_edit.h

```diff
@@ -42,6 +42,14 @@ class TextEdit : public Control {
 	GDCLASS(TextEdit, Control);
 
 public:
+	/* Edit Actions. */
```

Comment:
Unnecessary comment.

...although I see there are similar comments below :/
It only repeats enum name.